apps/api/src/routes/v1.ts

```diff
@@ -242,6 +244,18 @@ v1Router.get(
   wrap(extractStatusController),
 );
 
+v1Router.post(
+  "/generate-llmstxt",
+  authMiddleware(RateLimiterMode.Extract),
+  wrap(generateLLMsTextController),
+);
+
+v1Router.get(
+  "/generate-llmstxt/:jobId",
+  authMiddleware(RateLimiterMode.ExtractStatus),
+  wrap(generateLLMsTextStatusController),
+);
+
```

Comment:
should just be `/llmstxt` imo
```suggestion
v1Router.post(
  "/llmstxt",
  authMiddleware(RateLimiterMode.Extract),
  wrap(generateLLMsTextController),
);

v1Router.get(
  "/llmstxt/:jobId",
  authMiddleware(RateLimiterMode.ExtractStatus),
  wrap(generateLLMsTextStatusController),
);

```